dallinger/command_line/__init__.py

```diff
@@ -509,10 +506,17 @@ def hibernate(app):
         heroku_app.addon_destroy(addon)
 
 
-def _current_hits(service, app):
-    if app is not None:
-        return service.get_hits(hit_filter=lambda h: h.get("annotation") == app)
-    return service.get_hits()
+def get_recruiter(recruiter_name):
```

Comment:
I think `recruiter.by_name` is meant to serve the role of this function already?

Reply:
Great idea. I also added `kwargs` to the function to be able to pass `not_validate_config` (see comment below).